src/storage/index.ts

```diff
@@ -69,4 +70,4 @@ export class Storage<Scopes extends unknown[], Schema> {
  *
  *   `global.set(['doNotTrack'], true)`
  */
-export const device = new Storage<[], Device>({ id: 'global' });
+export const device = new Storage<[], Device>({ id: STORAGE_IDS.GLOBAL });
```

Comment:
Same idea: my idea was to have a single storage instance (or a couple), and clear _specific keys_ within them when needed.

Also, I'd prefer to not tie this new interface into the old `STORAGE_IDS` values. The keys for this storage instance live on the `Device` schema and can be cleared that way.